test/integration/sequelize.test.js

```diff
@@ -288,7 +294,7 @@ describe(Support.getTestDialectTeaser('Sequelize'), () => {
             ]
           }
         }))).to.be.rejectedWith(Sequelize.UniqueConstraintError).then(() => {
-          expect(spy.callCount).to.eql(3);
+          expect(spy.callCount).to.eql(dialect === 'db2' ? 1 : 3);
```

Comment:
Why is the behavior on Db2 different from other dialects here?

Reply:
@papb Seems retry is not working for db2 here. The control comes here after first UniqueConstaintError only.